robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -106,6 +151,38 @@ public class ShadowAccessibilityNodeInfo {
 
   private int textSelectionEnd = UNDEFINED_SELECTION_INDEX;
 
+  private boolean refreshReturnValue = true;
+
+  private int movementGranularities; //16
+
+  private CharSequence packageName; //14
+#if ($api >= 18)
+  private String viewIdResourceName; //18
+#end
+#if ($api >= 19)
+  private CollectionInfo collectionInfo; //19
+
+  private CollectionItemInfo collectionItemInfo; //19
+
+  private int inputType; //19
+
+  private int liveRegion; //19
+
+  private RangeInfo rangeInfo; //19
+#end
+#if ($api >= 21)
+  private int maxTextLength; //21
+
+  private CharSequence error; //21
+#end
+#if ($api >= 22)
+  private AccessibilityNodeInfo traversalAfter; //22
```

Comment:
I think traversalAfter and traversalBefore need to be accounted for in recycle()